tox.ini

```diff
@@ -50,8 +52,7 @@ deps:
     sphinx
 
 [flake8]
-enable-extensions = U4
 exclude = conf.py
-hang-closing = True
```

Comment:
Affects flake8's `E123` and `E133` rules, which are too-strict interpretations of PEP8. See review for `ignore` below.